test/build_container.sh

```diff
@@ -27,8 +27,14 @@ else
 fi
 
 pushd ./docker-kong
-  if [ "$RESTY_IMAGE_BASE" == "rhel" ]; then
-    sed -i.bak 's/^FROM .*/FROM 'registry.access.redhat.com\\/ubi${RESTY_IMAGE_TAG}\\/ubi'/' Dockerfile.$PACKAGE_TYPE
+  if \
+    [ "$RESTY_IMAGE_BASE" == 'rhel' ] || \
+    [[ "$RESTY_IMAGE_BASE" == *'/ubi'* ]] || \
+    [[ "$RESTY_IMAGE_BASE" == *'redhat'* ]] \
+  ; then
```

Comment:
nit: you can nix the `\` here and `;` on the next line:

```suggestion
    [[ "$RESTY_IMAGE_BASE" == *'redhat'* ]]
  then
```